docs/sources/next/javascript-api/k6-experimental/browser/mouse/down.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'down([options])'
+description: 'Mouse dispatches a `mousedown` event on the current position of the mouse.'
+---
+
+# down([options])
+
+Dispatches a `mousedown` event on the mouse's current position. Mouse can be moved to a different position using [`mouse.move()`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/mouse/move).
```

Comment:
Do we need to elaborate a little why a user might want to downClick and move (dragging an element i assume)? Otherwise it doesn't make sense to mention mouseMove here.

Reply:
It's related to [this issue](https://github.com/grafana/xk6-browser/issues/1317). Previously, `down` and `up` were accepting `x` and `y` coordinates, which was [incorrect](https://github.com/grafana/xk6-browser/pull/1318).

`down` and `up` are closely related to `move` because without a `move`, users will only be able to click (well, `down` or `up`) on the same part of the virtual screen. So, they first `move` and then use one of `down` or `up` to click to the `x` and `y` coordinates. That's why `move` is related.

But I can expand this explanation here.

Reply:
I retyped it like so [128c37e](https://github.com/grafana/k6-docs/pull/1594/commits/128c37ead172de3b7b540cb4870d38baa4a54365). It might be "slightly" better why these APIs are related.